app/views/hq/sponsors/show.html.haml

```diff
@@ -2,8 +2,12 @@
   %div.instance
     = link_to 'Edit Sponsor', edit_hq_sponsor_path(@sponsor.id),
       class: 'btn btn-default', role: 'button'
-    = link_to 'Link to Orphan', '#',
+    = link_to 'New Sponsor', new_hq_sponsor_path,
       class: 'btn btn-default', role: 'button'
+    - if @sponsor.status.name != 'Inactive' and @sponsor.status.name != 'On Hold'
```

Comment:
I think `if @sponsor.eligible_for_sponsorship?` would be better here.
Adding an `else` and show the button `disabled` would be useful